src/main/java/com/shepherdjerred/capstone/server/events/events/PlayerJoinEvent.java

```diff
@@ -3,14 +3,17 @@
 
 import com.shepherdjerred.capstone.common.player.Player;
 import com.shepherdjerred.capstone.events.Event;
+import com.shepherdjerred.capstone.server.network.ClientId;
 import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
 import lombok.Getter;
 import lombok.ToString;
 
 @Getter
 @ToString
+@EqualsAndHashCode
 @AllArgsConstructor
 public class PlayerJoinEvent implements Event {
-
+  private final ClientId clientId;
```

Comment:
A ClientId probably doesn't belong here, but it's not worth the work to find a better design.